unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/IdleChecker/Tests/IdleCheckerShould.cs

```diff
@@ -0,0 +1,39 @@
+using System.Collections;
+using System.Linq;
+using DCL;
+using DCL.Components;
+using DCL.Controllers;
+using DCL.Helpers;
+using DCL.Models;
+using NUnit.Framework;
+using UnityEngine;
+using UnityEngine.TestTools;
+
+namespace Tests
+{
+    public class IdleCheckerShould
+    {
+        private IIdleChecker idleChecker;
+
+        [Test(ExpectedResult = (IEnumerator)null)]
```

Comment:
```suggestion
        [UnityTest]
```
Please just use the `[UnityTest]` attribute here that's tailored for coroutines.